README.md

```diff
@@ -1 +1,38 @@
-# BlazorTesting
\ No newline at end of file
+# BlazorTesting
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [How to run the tests](#How-to-run-the-tests)
+
+## General info
+This project is sample example of how you could get started with testing a Blazor application by adding tests to the template blazor application.
```

Comment:
_This project contains a collection of samples to help you get started with testing a Blazor application by extending the template blazor application with tests._?

Reply:
I agree, I re-wrote this like 3 times, and couldn't get the wording quite right, I'll go with yours